.circleci/config.yml

```diff
@@ -38,8 +38,23 @@ workflows:
           requires:
             - build
 
+      # Push to Aliyun should execute for non-master branches only once manually approved.
       - architect/push-to-docker:
-          name: push-cluster-operator-to-aliyun
+          name: push-cluster-operator-to-aliyun-pr
+          image: "registry-intl.cn-shanghai.aliyuncs.com/giantswarm/cluster-operator"
+          username_envar: "ALIYUN_USERNAME"
+          password_envar: "ALIYUN_PASSWORD"
+          type: approval
```

Comment:
Does it work like that? I thought approval jobs are noop.

Reply:
You are right, fixed it, PTAL